test/e2e/operator_test.go

```diff
@@ -137,30 +146,49 @@ func TestDefaultContour(t *testing.T) {
 
 	// Create a sample workload for e2e testing.
 	appName := fmt.Sprintf("%s-%s", testAppName, testName)
-	if err := newDeployment(ctx, kclient, appName, specNs, testAppImage, testAppReplicas); err != nil {
-		t.Fatalf("failed to create deployment %s/%s: %v", specNs, appName, err)
+	if err := newDeployment(ctx, kclient, appName, cfg.SpecNs, testAppImage, testAppReplicas); err != nil {
```

Comment:
:+1: glad to see this being changed, I started down this path in one of my previous PR's but abandoned it